File: VB/EventToCommandExample/Models/SampleData.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq

Namespace EventToCommandExample.Models
	Public Class SampleData
		Private privateID As Integer
		Public Property ID() As Integer
			Get
				Return privateID
			End Get
			Set(ByVal value As Integer)
				privateID = value
			End Set
		End Property
		Private privateText As String
		Public Property Text() As String
			Get
				Return privateText
			End Get
			Set(ByVal value As String)
				privateText = value
			End Set
		End Property
		Private privateValue As Integer
		Public Property Value() As Integer
			Get
				Return privateValue
			End Get
			Set(ByVal value As Integer)
				privateValue = value
			End Set
		End Property
	End Class
End Namespace

File: CS/EventToCommandExample/Models/SampleData.cs
namespace EventToCommandExample.Models {
    public class SampleData {
        public virtual int ID { get; set; }
        public virtual string Text { get; set; }
        public virtual int Value { get; set; }
        public override string ToString() {
            return string.Format("{0} - {1} - {2}", ID, Text, Value);
        }
    }
}
